### ISSUE TITLE
M2 Apple

### ISSUE THREAD

**Issue by eduardocalfaia** [carlmontanari/boxen#47]
Hi guys,
does he tool can be used in M2 Apple architecture?

Thank you in advanced

BR
Eduardo

**Comment by carlmontanari** [comment]
Would boxen compile/run? It should, yes. Are there any plaforms you’d be able to package (without the magic of rosetta), probably no.

**Comment by eduardocalfaia** [comment]
> Would boxen compile/run? It should, yes. Are there any plaforms you’d be able to package (without the magic of rosetta), probably no.

yes, I have compiled and it is ok, but now I am experiment the error below:

~/boxen/bin <main> ./boxen package --disk ~/hda.qcow2
      info   1709421134 package requested for disk '/Users/eduardoalfaia/hda.qcow2'
  critical   1709421134 failed gleaning source data from disk '/Users/eduardoalfaia/hda.qcow2'
  critical   1709421134 error allocating disks for packaging: inspectionError: failed gleaning source data from disk '/Users/eduardoalfaia/hda.qcow2'
	🆘 finished unsuccessfully in 0 seconds
inspectionError: failed gleaning source data from disk '/Users/eduardoalfaia/hda.qcow2'

**Comment by hellt** [comment]
@eduardocalfaia that's because your file doesn't have any info as to which platform/version it belongs to.
You can use the cli flags to provide the platform information

**Comment by eduardocalfaia** [comment]
> @eduardocalfaia that's because your file doesn't have any info as to which platform/version it belongs to. You can use the cli flags to provide the platform information

Hi hellt, to PAN OS 8.0 which would be?
